src/components/CardSelectModal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { BANK_LIST } from '../utils/constants';
+import BankIcon from './BankIcon';
+import './CardSelectModal.css';
+
+type modalProps = {
+  determineCardType: React.Dispatch<React.SetStateAction<string>>;
+  closeModal: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const CardSelectModal = ({ determineCardType, closeModal }: modalProps) => {
+  return (
+    <>
+      <div
+        className="card-select-modal-backdrop"
+        onClick={() => {
+          closeModal(false);
+        }}
```

Comment:
다음과 같이 쓸 수 있을 것 같은데, 중괄호를 써준 이유가 있을까요?
```suggestion
        onClick={() => closeModal(false)}
```